__tests__/app.test.js

```diff
@@ -3,6 +3,7 @@ const request = require("supertest");
 const db = require("../db/connection");
 const testData = require("../db/data/test-data/index.js");
 const seed = require("../db/seeds/seed.js");
+const users = require("../db/data/test-data/users");

```

Comment:
you shouldn't need to require in the users data here
a) you can be free to hardcode any usernames necessary
b) if you wanted to access it, it's already inside the `testData` required in previously